vast.go

```diff
@@ -127,9 +127,9 @@ type Wrapper struct {
 	// xml element within the Extensions element.
 	Extensions []Extension `xml:"Extensions>Extension,omitempty"`
 
-	FallbackOnNoAd           bool `xml:"fallbackOnNoAd,attr,omitempty"`
-	AllowMultipleAds         bool `xml:"allowMultipleAds,attr,omitempty"`
-	FollowAdditionalWrappers bool `xml:"followAdditionalWrappers,attr,omitempty"`
+	FallbackOnNoAd           string `xml:"fallbackOnNoAd,attr,omitempty"`
+	AllowMultipleAds         string `xml:"allowMultipleAds,attr,omitempty"`
+	FollowAdditionalWrappers string `xml:"followAdditionalWrappers,attr,omitempty"`
```

Comment:
this looks good, @rs please merge once the above is moved out to a different PR.